docs/node-operators/staking-and-snarking.mdx

```diff
@@ -1,39 +1,64 @@
 
 ## Participating in Consensus
 
-The Mina network is secured by [Proof-of-Stake consensus](../glossary#proof-of-stake) With this model of consensus, you don't need to have complex equipment like in Bitcoin mining. By simply having MINA in our wallet, we can choose to either stake it ourselves, or delegate it to another node. Let's first see how to stake MINA ourselves:
+The Mina network is secured by [proof of stake (PoS)](/glossary#proof-of-stake-pos) consensus. With this model of consensus, you don't need to have complex equipment like in Bitcoin mining. By simply having MINA in your wallet, you can choose to stake your MINA or delegate it to another node. 
 
-:::caution
+Mina is a public and decentralized blockchain that is open for anyone in the world to participate in actively or passively. You can help increase the security of the network by becoming nodes or block producers, or you can help lower the cost of transactions by becoming SNARK producers, or you can be both.
 
-To properly remain synced to the network and participate in consensus, it is important that your server is running some form of a [clock synchronization protocol](https://en.wikipedia.org/wiki/Clock_synchronization). We recommend using [NTP](https://en.wikipedia.org/wiki/Network_Time_Protocol), which is relatively easy to setup and comes already installed as a default service on many popular linux distros.
+## Staking MINA
 
-:::
+You can earn block rewards by participating in block production through staking. 
 
-### Staking MINA
+- Your new stake delegation comes into effect after a latency period of a 2-4 weeks.
+- You can undelegate at any time with no penalty. Submit a delegation transaction with the new staking service or back to yourself if you want to be a block producer. Updates are made after a delay of 1-2 epochs.
 
-We can try out staking with our MINA by issuing the following command:
+### Requirements
 
-    mina client set-staking --public-key $MINA_PUBLIC_KEY
+To properly remain synced to the network and participate in consensus, your server must run a [clock synchronization protocol](https://en.wikipedia.org/wiki/Clock_synchronization).
 
-Alternatively, you can restart the daemon with the `-block-producer-pubkey` flag:
+Network Time Protocol [(NTP)](https://en.wikipedia.org/wiki/Network_Time_Protocol) is the most common method for synchronizing computer clocks over a network. 
+
+To synchronize your server software clock with internet time servers, be sure to set up NTP. Many popular Linux distributions include NTP as a default service.
+
+## Set or change the block producer key
+
+Use the `mina client delegate-stake` subcommand to set or change delegation of the block producer key. See the [Mina Client](/node-operators/mina-cli-reference#mina-client) CLI help.
```

Comment:
I think it is better to point to the `Delegating MINA` that follows this section and which has more information about delegating